code/modules/mob/living/simple_animal/revenant/revenant_abilities.dm

```diff
@@ -187,4 +206,66 @@
 					playsound(S, 'sound/machines/warning-buzzer.ogg', 50, 1)
 					new/obj/effect/overlay/temp/revenant(S.loc)
 					S.spark_system.start()
-					S.emp_act(1)
\ No newline at end of file
+					S.emp_act(1)
+
+//Blight: Infects nearby humans and in general messes living stuff up.
+/obj/effect/proc_holder/spell/aoe_turf/revenant/blight
+	name = "Blight"
+	desc = "Causes nearby living things to waste away."
+	charge_max = 200
+	range = 3
+	cast_amount = 50
+	unlock_amount = 200
+	action_icon_state = "blight"
+
+/obj/effect/proc_holder/spell/aoe_turf/revenant/blight/cast(list/targets, mob/living/simple_animal/revenant/user = usr)
+	if(attempt_cast(user))
+		for(var/turf/T in targets)
+			spawn(0)
+				for(var/mob/living/mob in T.contents)
+					if(mob == user)
+						return
```

Comment:
I don't know what continue does and it did affect all nearby mobs and glowshroom stuff when I tested it, is it a performance thing?